client/src/components/common/Header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import Button from '../Button/index';
+import './style.css';
+import '../Button/style.css';
+import Logo from './logo1.png';
+
+class Header extends Component {
+  state = {
+  login:false ,
+  join:false
+};
```

Comment:
State is unused. Changing the state from false to true requires a handler function.